### DISCUSSION TITLE
Is it possible to return log output with no color encoding?

### DISCUSSION THREAD

**Discussion by ShahBinoy** [slingdata-io/sling-cli#112] [Q&A] [answered]
Color encoded log statements make up for lot of extra processing on each line output. Is it possible to disable color encoding and just pump out raw text based logs

**Comment by flarco** [comment, marked as answer]
There is actually some legacy logic... Try settings env var `G_LOGGING=TASK` to disable color. Actually, also `G_LOGGING=WORKER` outputs without color in JSON format. Forgot about those. Let me update the logic for the env var to something like `SLING_LOGGING = JSON | NO_COLOR | DEFAULT` in the next release.
